source/default/_patterns/00-protons/legacy/css/components/UTC-custom-blocks/_utc_hover_images.css

```diff
@@ -0,0 +1,341 @@
+      aspect-ratio: 3/2!important;
+    }
+    /*center the vertical image*/
+    .image-output-horizontal .input-vertical img {
+      top: 50%;
+      transform: translateY(-50%);
+    }
+    .image-output-vertical .input-horizontal {
+      aspect-ratio: 2/3!important;
+    }
+    /*center the horizontal image*/
+    .image-output-vertical .input-horizontal img {
+      left: 50%;
+      transform: translateX(-50%);
+      max-width: unset;
+      width: unset;
+      height: 100%;
+    }
+
+/*additional reponsive styles*/
+@media (min-width: 768px) and (max-width: 1024px) {
+  figure.utc-hover-image-effect a {
+    @apply opacity-0 inline-block relative text-white px-4 py-1 font-bold text-center w-auto h-auto bg-utc-new-blue-500;
+    transform: translate3d(0%, 30%, 0);
+    transition-delay: 0s;
+  }
+  figure.utc-hover-image-effect:hover a {
+    @apply opacity-100;
+    transform: translate3d(0%, 0%, 0);
+    transition-delay: 0.8s;
+  }
+  .grid-cols-4 {
+    @apply grid-cols-2;
+  }
+}
+@media (max-width: 767px) {
+  .utc-image-hover {
+    display: block!important;
+    @apply text-center;
+  }
+  figure.utc-hover-image-effect {
+    @apply my-8 mx-auto float-none;
+  }
+  figure.input-horizontal {
+    @apply max-w-max;
+  }
+  figure.utc-hover-image-effect figcaption {
+    @apply text-left;
+  }
+  figure.utc-hover-image-effect a {
+    @apply opacity-0 inline-block mt-4 relative text-white px-4 py-1 font-bold text-center w-auto h-auto bg-utc-new-blue-500;
+    transform: translate3d(0%, 30%, 0);
+    transition-delay: 0s;
+  }
+  .image-output-vertical figure.utc-hover-image-effect a {
+    @apply py-2;
+  }
+  figure.utc-hover-image-effect:hover a {
+    @apply opacity-100;
+    transform: translate3d(0%, 0%, 0);
+    transition-delay: 0.8s;
+  }
+}
+@media (max-width: 459px) {
+  figure.utc-hover-image-effect h2 {
+    @apply text-xl;
+  }
+  figure.utc-hover-image-effect p {
+    @apply text-lg;
+  }
+  .hover-image-space {
+    @apply mt-0 h-0;
+  }
+}
+@media (max-width: 420px) {
+  figure.image-count-1:hover figcaption {
+    @apply p-8;
+  }
+  .hover-image-space {
+    @apply mt-10;
+  }
+}
+@media (max-width: 320px) {
+  figure.utc-hover-image-effect:hover figcaption,
+  figure.image-count-1:hover figcaption,
+  figure.image-count-2:hover figcaption {
+    @apply p-4;
+  }
+  figure.image-count-4:hover figcaption,
+  figure.image-count-5:hover figcaption {
+    @apply p-8;
+  }
+  figure.utc-hover-image-effect h2 {
+    @apply pb-0;
+  }
+}
+@media (max-width: 280px) {
+  figure.utc-hover-image-effect p {
+    @apply text-sm;
+  }
+  figure.image-count-4 p, 
+  figure.image-count-5 p {
+    @apply text-xs;
+  }
+  .image-output-horizontal .hover-image-space {
+    @apply mt-6;
+  }
+  .image-output-vertical figure.utc-hover-image-effect:hover figcaption {
+    @apply p-12;
+  }
+}
+
+
+/****Internal pages treatment***/
+.themag-layout--twocol-section--3-9 figure.utc-hover-image-effect figcaption {
```

Comment:
the challenge with this it assumes the 3-9 layout but they could flip it to 3-9 or in some cases is a 3 column layout.